01/app.js

```diff
@@ -0,0 +1,10 @@
+
+const user = {
+    firstName: 'Anna',
+    lastName: 'Kowalska',
+    sex: 'female',
+    age: '20',
+}
+
+
+    console.log(user);
```

Comment:
Pamiętaj, że `user.firstName` to pobranie konkretnej wartości dla wpisanej właściwości.